docs/Manual/08 Deferred Layout of Constant Arrays.md

```diff
@@ -2,20 +2,20 @@
 [//]: # (Version: v1.2.1)
 
 # Section 8: Deferred layout of constant arrays
-We revisit the topic of memory layout of constant arrays. As mentioned in [Section 1](<01%20Arrays.md>), the contents of constant arrays are known at compile time, they are immutable, and they are not externally visible. This allows Accera to store them using a non-standard layout, optimized for a specific plan. In some cases, there may even be a benefit to storing multiple copies of each array element (e.g., storing a matrix in both row-major and column-major layouts).
+Let's revisit the memory layout of constant arrays. As explained in [Section 1](<01%20Arrays.md>), the contents of constant arrays are known at compile-time, and these contents are immutable and hidden externally. Accera stores constant arrays in a non-standard memory layout optimized for a particular plan. In some cases, storing multiple copies of each array element may even prove advantageous (e.g., storing a matrix in row-major and column-major layouts).
 
 ## Deferred layout based on a cache
-An Accera cache strategy makes local copies of an array's active blocks. The constant array can be laid out based on a defined cache. Namely, the array is stored by serializing its active blocks one-after-the-other. If the caching strategy is `thrifty=True`, no data needs to be copied at runtime and the active blocks are ready to use.
+Accera cache strategy creates local copies of an array's active blocks. The constant array can be arranged based on the defined cache. Specifically, the array is stored by consecutively serializing the active blocks. If the caching strategy is `thrifty=True`, the active blocks are ready to use without copying the data.   
 
-To define an array layout based on a cache, note that the Accera DSL has to overcome a chicken-and-egg situation: on one hand, arrays need to be defined upfront, even before the nest logic; on the other hand, the array layout depends on a cache, which is only defined as part of the plan. We overcome this problem by splitting the array definition into two parts. We still define the constant array upfront, but avoid committing to a specific layout:
+To define an array layout based on the cache, Accera DSL has to overcome the chicken-and-egg paradox. While on the one hand, arrays need to be defined even before the nest logic. On the other hand, array layout depends on a cache, which is defined only as a part of a plan. In Accera, we overcome this situation by splitting the array definition into two parts. Though we still define the constant array upfront, we avoid committing to a specific layout. 
```

Comment:
"a cache" - we can have different cache configurations, so there can be multiple possible caches at different active blocks for a given array.